types/experiments.ts

```diff
@@ -34,10 +34,16 @@ export interface Experiment {
   features: Array<Feature>;
   /** Branch configuration for the experiment */
   branches: Array<Branch>;
-  /** Actual publish date of the experiment */
-  startDate: Date;
-  /** Actual end date of the experiment */
-  endDate: Date | null;
+  /**
+   * Actual publish date of the experiment
+   * @format date-time
+   */
+  startDate: string;
+  /**
+   * Actual end date of the experiment
+   * @format date-time
```

Comment:
Wow, cool!
I didn't even know we could define JSON schema properties using these [annotations](https://github.com/YousefED/typescript-json-schema#annotations).

Confirmed they passed through correctly by pulling this branch, running <kbd>npm run build</kbd> and then inspecting the generated schemas:

```js
          startDate: {
            type: "string",
            description: "Actual publish date of the experiment",
            format: "date-time",
          },
          endDate: {
            type: ["string", "null"],
            description: "Actual end date of the experiment",
            format: "date-time",
          },
```

Possibly worth filing a new issue for adding more annotations for numbers to cast to positive integers only, etc.